src/components/compound/DlPagination/components/PageNavigation.vue

```diff
@@ -212,6 +212,18 @@ export default defineComponent({
         },
         modelValue() {
             this.value = this.modelValue
+        },
+        maxDisplayRange() {
```

Comment:
Why is this needed?

Reply:
It was added so that UI immediately updates as user changes values in the demo page. I removed it now because it's not needed in prod since it is set only once.